info.json

```diff
@@ -12,6 +12,7 @@
     "(?) Dectorio",
     "stdlib >= 1.4.2",
     "(?) space-exploration",
+    "(?) Krastorio",
```

Comment:
This wasn't strictly needed. This mod loads in after krastorio just by virtue of its name being later in the alphabet. Still, doesn't hurt to add it as a dependency just to be sure.